src/regressor.jl

```diff
@@ -78,12 +78,17 @@ function MLJModelInterface.fit(model::Union{NeuralNetworkRegressor, Multivariate
 
     # When it has no categorical features
     n_input = Tables.schema(X).names |> length
-    n_output = length(y[1])
-    chain = fit(model.builder, n_input, n_output)
-
     data = collate(model, X, y, model.batch_size)
 
-    target_is_multivariate = y isa AbstractVector{<:Tuple}
+    target_is_multivariate = y isa Tables.MatrixTable
+    if target_is_multivariate
+        target_columns = Tables.schema(y).names
+    else
+        target_columns = [""] # We won't be using this
+    end
+
+    n_output = length(target_columns)
+    chain = fit(model.builder, n_input, n_output)
```

Comment:
This gives n_ouput = 0 in the univariate case. Don't you want this to be 1?

Reply:
@ablaom n_output will be 1 in this case, which is the length of [""].